app/crud.py

```
@@ -0,0 +1,152 @@
+    
+    Raises:
+        HTTPException: If an error occurs during database retrieval, with a 500 status code.
+    """
+    try:
+        result = await db.execute(select(ToDo))
+        return result.scalars().all()
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Error retrieving ToDos: {e}")
+
+
+async def get_todo(db: AsyncSession, todo_id: int):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_todo`

(ANN201)